src/ui/preferences-window/tabs/ControlsTab.swift

```diff
@@ -41,14 +47,23 @@ class ControlsTab {
         let miscCheckboxes = StackView([StackView(enableCursorFollowFocus)], .vertical)
         let shortcuts = StackView([focusWindowShortcut, previousWindowShortcut, cancelShortcut, closeWindowShortcut, minDeminWindowShortcut, quitAppShortcut, hideShowAppShortcut].map { (view: [NSView]) in StackView(view) }, .vertical)
         let orPress = LabelAndControl.makeLabel(NSLocalizedString("While open, press:", comment: ""), shouldFit: false)
-        let (holdShortcut, nextWindowShortcut, tab1View) = toShowSection("")
-        let (holdShortcut2, nextWindowShortcut2, tab2View) = toShowSection("2")
-        let tabView = TabView([(NSLocalizedString("Shortcut 1", comment: ""), tab1View), (NSLocalizedString("Shortcut 2", comment: ""), tab2View)])
+        let (holdShortcut, nextWindowShortcut, tab1View) = toShowSection(0)
+        let (holdShortcut2, nextWindowShortcut2, tab2View) = toShowSection(1)
+        let (holdShortcut3, nextWindowShortcut3, tab3View) = toShowSection(2)
+        let (holdShortcut4, nextWindowShortcut4, tab4View) = toShowSection(3)
+        let (holdShortcut5, nextWindowShortcut5, tab5View) = toShowSection(4)
+        let tabView = TabView([
+            (NSLocalizedString("Shortcut 1", comment: ""), tab1View),
+            (NSLocalizedString("Shortcut 2", comment: ""), tab2View),
+            (NSLocalizedString("Shortcut 3", comment: ""), tab3View),
+            (NSLocalizedString("Shortcut 4", comment: ""), tab4View),
+            (NSLocalizedString("Shortcut 5", comment: ""), tab5View),
```

Comment:
I thought we may use a variable like `"Shortcut %@"` here to avoid having each language translate 5 times the same thing, but then I checked, and some languages wouldn't work with this pattern (e.g. `"Shortcut 2" = "الإختصار ٢";`). Oh well 😅